test/uk/gov/hmrc/essttpstubs/controllers/ArrangementControllerSpec.scala

```diff
@@ -52,6 +52,7 @@ class ArrangementControllerSpec extends ItSpec {
           |	}],
           |	"channelIdentifier": "eSSTTP",
           |	"regimeType": "PAYE",
+          | "regimePaymentFrequency": "Monthly",
```

Comment:
just to double check, are they definitely expecting upper case `M` and not lower case `m`?

Reply:
Yeah

Reply:
I think it was finalised that it's always uppercased

Reply:
waiiit, we have a mix, let me check

Reply:
In stubs we have `Monthly` and frontend it's `monthly`. - I don't think it matters? But we should make consistent

Reply:
In stubs we have `Monthly` and frontend it's `monthly`. - I don't think it matters? But we should make consistent

Reply:
is it the relating to the same call in the FE and stub where it's different? Consistency would be better yeah if possible

Reply:
it's bit badly formatted at the moment

Reply:
Not sure why github is displaying it like that, it's got the correct spacing in intellij,
but I've updated to be monthly